lib/src/components/NxNumberInput/stateHelpers.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { StateProps, Validator } from './types';
+import { curryN, trim } from 'ramda';
+
+/**
+ * @return an initialized state with the specified value and isPristine set to true.
+ */
+export function initialState(value: string, validator?: Validator): StateProps {
```

Comment:
It would probably make things easier downstrean if the `value` parameter here could be a string or a number

Reply:
> It would probably make things easier downstrean if the `value` parameter here could be a string or a number

Done here: https://github.com/sonatype/sonatype-react-shared-components/pull/486/commits/851483d1abbace00aa442ad4812e8624005ccfc9